app/assess/data.py

```diff
@@ -815,6 +815,28 @@ def get_default_round_data():
     return round_response
 
 
+def get_tag(fund_id, round_id, tag_id) -> Tag:
+    endpoint = Config.ASSESSMENT_GET_TAG_ENDPOINT.format(
+        fund_id=fund_id, round_id=round_id, tag_id=tag_id
+    )
+    response = get_data(endpoint)
+    if response:
+        return Tag.from_dict(response)
+    return None
+
+
+def update_tag(fund_id: str, round_id: str, updated_tag: Dict) -> Tag:
+    endpoint = Config.ASSESSMENT_UPDATE_TAGS_ENDPOINT.format(
+        fund_id=fund_id, round_id=round_id
+    )
+    response = requests.put(url=endpoint, json=updated_tag)
+    if response.status_code == 200:
+        return response.json()[0]
+
+    current_app.logger.error(f"Unable to update tag: {updated_tag}")
+    return None
```

Comment:
Method signature is a little wrong here with type annotations, it looks like we provide a list of dictionaries: 

https://github.com/communitiesuk/funding-service-design-assessment/blob/bb50c5e6ba2033d4c736158ce865fdde7d420616/app/assess/tag_routes.py#L285-L286

Also, maybe instead of passing an array into the method, we do pass in just a dict and wrap it in an array before utilising the endpoint? Would save any other usages of the `update_tag` method from having to nested it in a list. 

i.e 
```suggestion
def update_tag(fund_id: str, round_id: str, updated_tag: Dict) -> Tag:
    endpoint = Config.ASSESSMENT_UPDATE_TAGS_ENDPOINT.format(
        fund_id=fund_id, round_id=round_id
    )
    response = requests.put(url=endpoint, json=[updated_tag])
    if response.status_code == 200:
        return response.json()[0]

    current_app.logger.error(f"Unable to update tag: {updated_tag}")
    return None
```

Reply:
Took me a while to figure out what was going here... which goes to show how confusing I'd made it 😆